traffic_ops/testing/api/v4/regions_test.go

```diff
@@ -175,15 +222,69 @@ func UpdateTestRegions(t *testing.T) {
 	if err != nil {
 		t.Errorf("cannot UPDATE Region by id: %v - %v", err, alert)
 	}
-
 }
 
-func GetTestRegions(t *testing.T) {
-	for _, region := range testData.Regions {
-		resp, _, err := TOSession.GetRegionByName(region.Name, nil)
-		if err != nil {
-			t.Errorf("cannot GET Region by region: %v - %v", err, resp)
-		}
+func VerifyPaginationSupportRegion(t *testing.T) {
+	qparams := url.Values{}
+	qparams.Set("orderby", "id")
+	regions, _, err := TOSession.GetRegions(qparams, nil)
```

Comment:
This test contains many lines that will segfault if the length of `regions` is not at least 3 - which it should check for

Reply:
Fixed